qemu/tests/zero_copy.py

```diff
@@ -52,6 +52,9 @@ def enable_zerocopytx_in_host(enable=True):
         enable_zerocopytx_in_host(False)
 
     error.context("Boot vm with 'vhost=on'", logging.info)
+    if params.get("nettype") == "user":
+        raise error.TestNAError("Unable start test with user networking,"
```

Comment:
As far as i know, no plan to fix the existing tests, only use the new method in new code.